package.json

```diff
@@ -12,15 +12,31 @@
     "spotify-web-api-node": "^4.0.0"
   },
   "scripts": {
-    "local": "tsc && node dist/app.js",
+    "lint": "tslint -t verbose --fix 'src/**/*.ts?(x)'",
+    "local": "nodemon src/app.ts",
     "start": "node dist/app.js",
     "build": "tsc"
   },
   "devDependencies": {
     "@types/express": "^4.17.2",
     "@types/node": "^12.12.6",
     "@types/spotify-web-api-node": "^4.0.1",
+    "husky": "^3.0.9",
+    "lint-staged": "^9.4.2",
+    "nodemon": "^1.19.4",
+    "ts-node": "^8.4.1",
     "tslint": "^5.20.1",
     "typescript": "^3.7.2"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
this will make it so you can't commit code that doesn't lint?

not sure if i love that but it's good for now. i'd rather just have a command to run lint locally, but make it check and enforce it on the PR

Reply:
If you have to run a linter to merge a PR, why don't you want it to run on a commit?